src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Nouns.cs

```diff
@@ -22,42 +22,47 @@ internal static class Nouns
         /// <summary>
         /// The prefix for cmdlet names.
         /// </summary>
-        private const string Prefix = "AzureRM";
+        private const string Prefix = "AzS";
```

Comment:
The reason is that the next word is going to be Capitalized, so it will always be AzSVM  (for example - there is no visual break between the S and the next noun.  This is why we use AzureRm instead of AzureRM in cmdlet names (but AzureRM in module names, where this doesn't apply)